tests/test_transpile.py

```diff
@@ -183,7 +183,7 @@ def test_identity(self):
     def test_partial(self):
         with open(os.path.join(self.fixtures_dir, 'partial.sql'), encoding='utf-8') as f:
             for sql in f:
-                self.assertEqual(transpile(sql, error_level=ErrorLevel.IGNORE)[0], sql.strip())
+                self.assertEqual(len(transpile(sql, error_level=ErrorLevel.IGNORE)), 0)
```

Comment:
why was this changed?

Reply:
Please see my [comment](https://github.com/tobymao/sqlglot/pull/25#discussion_r714214201). Basically since we now treat all `None` arguments as missing parsing of badly formed expressions now fails.

Reply:
ahh... that's a bit of a bummer that that's no longer support, since someone requested this feature

is it possible to allow this to happen (through some flag) -- the use case was so that we could parse partial expressions?